editor/components/skip-to-selected-block/index.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Aside: If "Skip To" buttons are expected to be commonplace, could make for a good base `components` component.

Reply:
Do you mean moving it to the `/components` directory?

Reply:
Yes, maybe not `SkipToSelectedBlock` as-is, since it an implementation specific to block usage, but maybe a `SkipTo`, which `SkipToSelectedBlock` can extend (or just inline `SkipTo` where we're using `SkipToSelectedBlock`).

Reply:
Ah OK. Hm at the moment I don't see the need for other "Skip To"s, the ones in core and this one should suffice, for now. We can always iterate later, if need be?